src/webapp/models.py

```diff
@@ -78,6 +78,9 @@ def __init__(self, config=None, strict=False):
         self.ligo_ldap_passfile = config.get("LIGO_LDAP_PASSFILE")
         self.ligo_ldap_url = config.get("LIGO_LDAP_URL")
         self.ligo_ldap_user = config.get("LIGO_LDAP_USER")
+        self.github_oauth_client_secret = config.get("GITHUB_OAUTH_CLIENT_SECRET")
+        self.osg_bot_github_token = config.get("OSG_BOT_GITHUB_TOKEN")
```

Comment:
Is this `OSG_BOT_GITHUB_TOKEN` different from the `WEBHOOK_GH_API_USER` / `WEBHOOK_GH_API_TOKEN` a little bit above?  (The `osg-bot` user uses these for reviewing/commenting/merging PRs.)

If they are different i wonder if there's a way to come up with a clearer naming scheme here to distinguish them.

Reply:
If i'm understanding the code right i think you are using a hard-coded `osg-doc-bot` user along with your `OSG_BOT_GITHUB_TOKEN` ?

If so then maybe to fit the model you could make that configurable also and use `DOC_GH_API_USER` / `DOC_GH_API_TOKEN` (or some other prefix that captures what this bot user is for).

Reply:
maybe `FORM_*` ...